Remotion/Development/Web/UnitTesting/Resources/FakeCacheableResourcePathBuilder.cs

```diff
@@ -0,0 +1,42 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+// 
+// The re-motion Core Framework is free software; you can redistribute it 
+// and/or modify it under the terms of the GNU Lesser General Public License 
+// as published by the Free Software Foundation; either version 2.1 of the 
+// License, or (at your option) any later version.
+// 
+// re-motion is distributed in the hope that it will be useful, 
+// but WITHOUT ANY WARRANTY; without even the implied warranty of 
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
+// GNU Lesser General Public License for more details.
+// 
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+// 
+using System;
+using Remotion.Web.Resources;
+
+namespace Remotion.Development.Web.UnitTesting.Resources
+{
+  /// <summary>
+  /// Fake implementation of the <see cref="ICacheableResourcePathBuilder"/> interface, intended for use in unit testing.
+  /// </summary>
+  public class FakeCacheableResourcePathBuilder : ResourcePathBuilderBase, ICacheableResourcePathBuilder
+  {
+    public FakeCacheableResourcePathBuilder ()
+        : base(new NullStaticResourceCacheKeyProvider())
+    {
+    }
```

Comment:
without checking the rest of the code: shouldn't we use a FakeStaticResourceCacheKeyProvider here so the FakeCachable is actually having an effect beyond the interface? It might be relevant in certain test scenarios to be able to distinuish the output of the two PathBuilder types?

Reply:
I looked at the usages again and decided to remove the fake entirely. It was only used because a `ICachedResourcePathBuilder` reference was necessary. I replaced the uses with a stub.